rollup.config.js

```diff
@@ -41,7 +41,7 @@ export default {
     name: moduleName,
     format: 'umd',
     exports: 'default',
-    sourcemap: false,
+    sourcemap: true,
```

Comment:
why do we change sourcemap in this branch?

Reply:
No I just have that when developing, will change it